docs: Update quick-start.md (#1667)
goreleaser/goreleaser: www/docs/quick-start.md

```

You can [customize](/customization) the generated `.goreleaser.yml` or leave

> it as-is, it's up to you. Note that the `.goreleaser.yml` needs to be checked into the source control.

Reviewer: Strictly speaking it could be in `.gitignore` which also avoids the problem with dirty git state, but I'd say that's in most cases bad idea and there should be almost no reason not to check it in. Sensitive data can be avoided via ENV variables, which is something users should use either way, whether it's OSS or a private project.
```suggestion
it as-is, it's up to you. It is best practice to check `.goreleaser.yml` into the source control.
```
